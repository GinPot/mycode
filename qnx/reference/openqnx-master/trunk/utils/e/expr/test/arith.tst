expr 0 '+' 0
expr 0 '+' 1
expr 0 '+' 10
expr 0 '+' -1
expr 0 '+' -3
expr 0 '-' 0
expr 0 '-' 1
expr 0 '-' 10
expr 0 '-' -1
expr 0 '-' -3
expr 0 '*' 0
expr 0 '*' 1
expr 0 '*' 10
expr 0 '*' -1
expr 0 '*' -3
expr 0 '/' 0
expr 0 '/' 1
expr 0 '/' 10
expr 0 '/' -1
expr 0 '/' -3
expr 0 '%' 0
expr 0 '%' 1
expr 0 '%' 10
expr 0 '%' -1
expr 0 '%' -3
expr 1 '+' 0
expr 1 '+' 1
expr 1 '+' 10
expr 1 '+' -1
expr 1 '+' -3
expr 1 '-' 0
expr 1 '-' 1
expr 1 '-' 10
expr 1 '-' -1
expr 1 '-' -3
expr 1 '*' 0
expr 1 '*' 1
expr 1 '*' 10
expr 1 '*' -1
expr 1 '*' -3
expr 1 '/' 0
expr 1 '/' 1
expr 1 '/' 10
expr 1 '/' -1
expr 1 '/' -3
expr 1 '%' 0
expr 1 '%' 1
expr 1 '%' 10
expr 1 '%' -1
expr 1 '%' -3
expr 10 '+' 0
expr 10 '+' 1
expr 10 '+' 10
expr 10 '+' -1
expr 10 '+' -3
expr 10 '-' 0
expr 10 '-' 1
expr 10 '-' 10
expr 10 '-' -1
expr 10 '-' -3
expr 10 '*' 0
expr 10 '*' 1
expr 10 '*' 10
expr 10 '*' -1
expr 10 '*' -3
expr 10 '/' 0
expr 10 '/' 1
expr 10 '/' 10
expr 10 '/' -1
expr 10 '/' -3
expr 10 '%' 0
expr 10 '%' 1
expr 10 '%' 10
expr 10 '%' -1
expr 10 '%' -3
expr -1 '+' 0
expr -1 '+' 1
expr -1 '+' 10
expr -1 '+' -1
expr -1 '+' -3
expr -1 '-' 0
expr -1 '-' 1
expr -1 '-' 10
expr -1 '-' -1
expr -1 '-' -3
expr -1 '*' 0
expr -1 '*' 1
expr -1 '*' 10
expr -1 '*' -1
expr -1 '*' -3
expr -1 '/' 0
expr -1 '/' 1
expr -1 '/' 10
expr -1 '/' -1
expr -1 '/' -3
expr -1 '%' 0
expr -1 '%' 1
expr -1 '%' 10
expr -1 '%' -1
expr -1 '%' -3
expr -3 '+' 0
expr -3 '+' 1
expr -3 '+' 10
expr -3 '+' -1
expr -3 '+' -3
expr -3 '-' 0
expr -3 '-' 1
expr -3 '-' 10
expr -3 '-' -1
expr -3 '-' -3
expr -3 '*' 0
expr -3 '*' 1
expr -3 '*' 10
expr -3 '*' -1
expr -3 '*' -3
expr -3 '/' 0
expr -3 '/' 1
expr -3 '/' 10
expr -3 '/' -1
expr -3 '/' -3
expr -3 '%' 0
expr -3 '%' 1
expr -3 '%' 10
expr -3 '%' -1
expr -3 '%' -3
