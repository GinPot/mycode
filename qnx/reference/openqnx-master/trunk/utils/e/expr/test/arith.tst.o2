0
1
10
-1
-3
0
-1
-10
1
3
0
0
0
0
0
expr: division by zero
0
0
0
0
expr: division by zero
0
0
0
0
1
2
11
0
-2
1
0
-9
2
4
0
1
10
-1
-3
expr: division by zero
1
0
-1
0
expr: division by zero
0
1
0
1
10
11
20
9
7
10
9
0
11
13
0
10
100
-10
-30
expr: division by zero
10
1
-10
-3
expr: division by zero
0
0
0
1
-1
0
9
-2
-4
-1
-2
-11
0
2
0
-1
-10
1
3
expr: division by zero
-1
0
1
0
expr: division by zero
0
-1
0
-1
-3
-2
7
-4
-6
-3
-4
-13
-2
0
0
-3
-30
3
9
expr: division by zero
-3
0
3
1
expr: division by zero
0
-3
0
0
