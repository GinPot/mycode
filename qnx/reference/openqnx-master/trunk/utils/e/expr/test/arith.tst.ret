expr 0 '+' 0 > /dev/null; echo Retval = $?
expr 0 '+' 1 > /dev/null; echo Retval = $?
expr 0 '+' 10 > /dev/null; echo Retval = $?
expr 0 '+' -1 > /dev/null; echo Retval = $?
expr 0 '+' -3 > /dev/null; echo Retval = $?
expr 0 '-' 0 > /dev/null; echo Retval = $?
expr 0 '-' 1 > /dev/null; echo Retval = $?
expr 0 '-' 10 > /dev/null; echo Retval = $?
expr 0 '-' -1 > /dev/null; echo Retval = $?
expr 0 '-' -3 > /dev/null; echo Retval = $?
expr 0 '*' 0 > /dev/null; echo Retval = $?
expr 0 '*' 1 > /dev/null; echo Retval = $?
expr 0 '*' 10 > /dev/null; echo Retval = $?
expr 0 '*' -1 > /dev/null; echo Retval = $?
expr 0 '*' -3 > /dev/null; echo Retval = $?
expr 0 '/' 0 > /dev/null; echo Retval = $?
expr 0 '/' 1 > /dev/null; echo Retval = $?
expr 0 '/' 10 > /dev/null; echo Retval = $?
expr 0 '/' -1 > /dev/null; echo Retval = $?
expr 0 '/' -3 > /dev/null; echo Retval = $?
expr 0 '%' 0 > /dev/null; echo Retval = $?
expr 0 '%' 1 > /dev/null; echo Retval = $?
expr 0 '%' 10 > /dev/null; echo Retval = $?
expr 0 '%' -1 > /dev/null; echo Retval = $?
expr 0 '%' -3 > /dev/null; echo Retval = $?
expr 1 '+' 0 > /dev/null; echo Retval = $?
expr 1 '+' 1 > /dev/null; echo Retval = $?
expr 1 '+' 10 > /dev/null; echo Retval = $?
expr 1 '+' -1 > /dev/null; echo Retval = $?
expr 1 '+' -3 > /dev/null; echo Retval = $?
expr 1 '-' 0 > /dev/null; echo Retval = $?
expr 1 '-' 1 > /dev/null; echo Retval = $?
expr 1 '-' 10 > /dev/null; echo Retval = $?
expr 1 '-' -1 > /dev/null; echo Retval = $?
expr 1 '-' -3 > /dev/null; echo Retval = $?
expr 1 '*' 0 > /dev/null; echo Retval = $?
expr 1 '*' 1 > /dev/null; echo Retval = $?
expr 1 '*' 10 > /dev/null; echo Retval = $?
expr 1 '*' -1 > /dev/null; echo Retval = $?
expr 1 '*' -3 > /dev/null; echo Retval = $?
expr 1 '/' 0 > /dev/null; echo Retval = $?
expr 1 '/' 1 > /dev/null; echo Retval = $?
expr 1 '/' 10 > /dev/null; echo Retval = $?
expr 1 '/' -1 > /dev/null; echo Retval = $?
expr 1 '/' -3 > /dev/null; echo Retval = $?
expr 1 '%' 0 > /dev/null; echo Retval = $?
expr 1 '%' 1 > /dev/null; echo Retval = $?
expr 1 '%' 10 > /dev/null; echo Retval = $?
expr 1 '%' -1 > /dev/null; echo Retval = $?
expr 1 '%' -3 > /dev/null; echo Retval = $?
expr 10 '+' 0 > /dev/null; echo Retval = $?
expr 10 '+' 1 > /dev/null; echo Retval = $?
expr 10 '+' 10 > /dev/null; echo Retval = $?
expr 10 '+' -1 > /dev/null; echo Retval = $?
expr 10 '+' -3 > /dev/null; echo Retval = $?
expr 10 '-' 0 > /dev/null; echo Retval = $?
expr 10 '-' 1 > /dev/null; echo Retval = $?
expr 10 '-' 10 > /dev/null; echo Retval = $?
expr 10 '-' -1 > /dev/null; echo Retval = $?
expr 10 '-' -3 > /dev/null; echo Retval = $?
expr 10 '*' 0 > /dev/null; echo Retval = $?
expr 10 '*' 1 > /dev/null; echo Retval = $?
expr 10 '*' 10 > /dev/null; echo Retval = $?
expr 10 '*' -1 > /dev/null; echo Retval = $?
expr 10 '*' -3 > /dev/null; echo Retval = $?
expr 10 '/' 0 > /dev/null; echo Retval = $?
expr 10 '/' 1 > /dev/null; echo Retval = $?
expr 10 '/' 10 > /dev/null; echo Retval = $?
expr 10 '/' -1 > /dev/null; echo Retval = $?
expr 10 '/' -3 > /dev/null; echo Retval = $?
expr 10 '%' 0 > /dev/null; echo Retval = $?
expr 10 '%' 1 > /dev/null; echo Retval = $?
expr 10 '%' 10 > /dev/null; echo Retval = $?
expr 10 '%' -1 > /dev/null; echo Retval = $?
expr 10 '%' -3 > /dev/null; echo Retval = $?
expr -1 '+' 0 > /dev/null; echo Retval = $?
expr -1 '+' 1 > /dev/null; echo Retval = $?
expr -1 '+' 10 > /dev/null; echo Retval = $?
expr -1 '+' -1 > /dev/null; echo Retval = $?
expr -1 '+' -3 > /dev/null; echo Retval = $?
expr -1 '-' 0 > /dev/null; echo Retval = $?
expr -1 '-' 1 > /dev/null; echo Retval = $?
expr -1 '-' 10 > /dev/null; echo Retval = $?
expr -1 '-' -1 > /dev/null; echo Retval = $?
expr -1 '-' -3 > /dev/null; echo Retval = $?
expr -1 '*' 0 > /dev/null; echo Retval = $?
expr -1 '*' 1 > /dev/null; echo Retval = $?
expr -1 '*' 10 > /dev/null; echo Retval = $?
expr -1 '*' -1 > /dev/null; echo Retval = $?
expr -1 '*' -3 > /dev/null; echo Retval = $?
expr -1 '/' 0 > /dev/null; echo Retval = $?
expr -1 '/' 1 > /dev/null; echo Retval = $?
expr -1 '/' 10 > /dev/null; echo Retval = $?
expr -1 '/' -1 > /dev/null; echo Retval = $?
expr -1 '/' -3 > /dev/null; echo Retval = $?
expr -1 '%' 0 > /dev/null; echo Retval = $?
expr -1 '%' 1 > /dev/null; echo Retval = $?
expr -1 '%' 10 > /dev/null; echo Retval = $?
expr -1 '%' -1 > /dev/null; echo Retval = $?
expr -1 '%' -3 > /dev/null; echo Retval = $?
expr -3 '+' 0 > /dev/null; echo Retval = $?
expr -3 '+' 1 > /dev/null; echo Retval = $?
expr -3 '+' 10 > /dev/null; echo Retval = $?
expr -3 '+' -1 > /dev/null; echo Retval = $?
expr -3 '+' -3 > /dev/null; echo Retval = $?
expr -3 '-' 0 > /dev/null; echo Retval = $?
expr -3 '-' 1 > /dev/null; echo Retval = $?
expr -3 '-' 10 > /dev/null; echo Retval = $?
expr -3 '-' -1 > /dev/null; echo Retval = $?
expr -3 '-' -3 > /dev/null; echo Retval = $?
expr -3 '*' 0 > /dev/null; echo Retval = $?
expr -3 '*' 1 > /dev/null; echo Retval = $?
expr -3 '*' 10 > /dev/null; echo Retval = $?
expr -3 '*' -1 > /dev/null; echo Retval = $?
expr -3 '*' -3 > /dev/null; echo Retval = $?
expr -3 '/' 0 > /dev/null; echo Retval = $?
expr -3 '/' 1 > /dev/null; echo Retval = $?
expr -3 '/' 10 > /dev/null; echo Retval = $?
expr -3 '/' -1 > /dev/null; echo Retval = $?
expr -3 '/' -3 > /dev/null; echo Retval = $?
expr -3 '%' 0 > /dev/null; echo Retval = $?
expr -3 '%' 1 > /dev/null; echo Retval = $?
expr -3 '%' 10 > /dev/null; echo Retval = $?
expr -3 '%' -1 > /dev/null; echo Retval = $?
expr -3 '%' -3 > /dev/null; echo Retval = $?
