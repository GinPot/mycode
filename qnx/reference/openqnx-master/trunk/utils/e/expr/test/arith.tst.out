0
1
10
-1
-3
0
-1
-10
1
3
0
0
0
0
0
Floating point exception 
0
0
0
0
Floating point exception 
0
0
0
0
1
2
11
0
-2
1
0
-9
2
4
0
1
10
-1
-3
Floating point exception 
1
0
-1
0
Floating point exception 
0
1
0
1
10
11
20
9
7
10
9
0
11
13
0
10
100
-10
-30
Floating point exception 
10
1
-10
-3
Floating point exception 
0
0
0
1
-1
0
9
-2
-4
-1
-2
-11
0
2
0
-1
-10
1
3
Floating point exception 
-1
0
1
0
Floating point exception 
0
-1
0
-1
-3
-2
7
-4
-6
-3
-4
-13
-2
0
0
-3
-30
3
9
Floating point exception 
-3
0
3
1
Floating point exception 
0
-3
0
0
